ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -35,32 +35,33 @@ class VPoolTester(CIConstants):
     LOGGER = Logger('scenario-{0}'.format(TEST_NAME))
 
     def __init__(self):
-        self.LOGGER.info("Initializing concurrent vpool testing.")
-        self.valid_storagerouters = []
+        VPoolTester.LOGGER.info("Initializing concurrent vpool testing.")
```

Comment:
self is valid here and should thus be used instead of VPoolTester as this will reference the static class (might be problematic when resetting variables for example)